src/scss/_statusbar.scss

```diff
@@ -134,8 +134,8 @@
         background-color: $color-cornflower-blue;
         color: $color-white;
 
-        &:hover { 
-          background-color: darken($color-cornflower-blue, 20%); 
+        &:hover {
```

Comment:
Should I to remove this change? I don't know how it happend, maybe Atom removed whitespace at line end when I saved it on previous commits.